File: Java/switchcaseatm.java
import java.util.Scanner;
public class switchcaseatm {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String nama;
        do {
            System.out.print("Masukkan nama: ");
            nama = sc.nextLine();
            System.out.print("Apakah nama sudah benar? (TRUE/FALSE): ");
        } while (!sc.nextLine().equalsIgnoreCase("TRUE"));

        System.out.print("Masukkan NIM (angka saja): ");
        long saldo = Long.parseLong(sc.nextLine());
        System.out.println("Halo " + nama + ", saldo awal kamu: Rp " + saldo);

        while (true) {
            System.out.println("\n===== MENU ATM =====");
            System.out.println("1. Cek Saldo\n2. Tarik Tunai\n3. Setor Tunai\n4. Transfer\n5. Keluar");
            System.out.print("Pilih menu: ");
            String pilih = sc.nextLine();

            switch (pilih) {
                case "1": System.out.println("Saldo: Rp " + saldo); break;
                case "2": System.out.print("Nominal tarik: "); saldo -= Long.parseLong(sc.nextLine()); System.out.println("Saldo: Rp " + saldo); break;
                case "3": System.out.print("Nominal setor: "); saldo += Long.parseLong(sc.nextLine()); System.out.println("Saldo: Rp " + saldo); break;
                case "4": System.out.print("Nominal transfer: "); saldo -= Long.parseLong(sc.nextLine()); System.out.println("Saldo: Rp " + saldo); break;
                case "5": System.out.println("Terima kasih " + nama); return;
                default: System.out.println("Pilihan tidak valid!");
            }
        }
    }
}

File: PHP/switchcaseatm.php
<?php
echo "Masukkan nama: ";
$nama = trim(fgets(STDIN));
echo "Apakah nama sudah benar? (TRUE/FALSE): ";
$cek = trim(fgets(STDIN));
while (strtoupper($cek) != "TRUE") {
    echo "Masukkan nama lagi: ";
    $nama = trim(fgets(STDIN));
    echo "Apakah nama sudah benar? (TRUE/FALSE): ";
    $cek = trim(fgets(STDIN));
}

echo "Masukkan NIM (angka saja): ";
$nim = trim(fgets(STDIN));
$saldo = (int)$nim;
echo "Halo $nama, saldo awal kamu: Rp $saldo\n";

while (true) {
    echo "\n===== MENU ATM =====\n";
    echo "1. Cek Saldo\n2. Tarik Tunai\n3. Setor Tunai\n4. Transfer\n5. Keluar\n";
    echo "Pilih menu: ";
    $pil = trim(fgets(STDIN));

    switch ($pil) {
        case "1": echo "Saldo: Rp $saldo\n"; break;
        case "2": echo "Nominal tarik: "; $tarik = (int)trim(fgets(STDIN)); $saldo -= $tarik; echo "Saldo: Rp $saldo\n"; break;
        case "3": echo "Nominal setor: "; $setor = (int)trim(fgets(STDIN)); $saldo += $setor; echo "Saldo: Rp $saldo\n"; break;
        case "4": echo "Nominal transfer: "; $tf = (int)trim(fgets(STDIN)); $saldo -= $tf; echo "Saldo: Rp $saldo\n"; break;
        case "5": echo "Terima kasih $nama!\n"; exit;
        default: echo "Pilihan tidak valid!\n";
    }
}